rwt_robot_monitor/www/index.html

```diff
@@ -37,17 +37,17 @@ <h2>All</h2>
           </div>
         </div>
         <div class="footer-stats">
-          <p>Last message received <span id="last-message-sec">0</span> secnods ago
+          <p>Last message received <span id="last-message-sec">0</span> seconds ago
         </div>
       </div>
     </div>
-    <script type="text/javascript" src="/rwt_plot/flot/jquery.min.js"></script>
+    <script type="text/javascript" src="../rwt_utils_3rdparty/jquery/jquery.min.js"></script>
```

Comment:
why this has to be `src="../rwt_utils_3rdparty/jquery/jquery.min.js"` and next line is `src="/rwt_utils_3rdparty/bootstrap/js/bootstrap.min.js"` ?